frontend/src/components/activePage/TableInstance/RowInstanceActions/RowInstanceActionsExtended.tsx

```diff
@@ -54,14 +53,14 @@ const RowInstanceActionsExtended: FC<IRowInstanceActionsExtendedProps> = ({
             <InfoOutlined />
           </Button>
         </Popover>
-        {ssh && (
+        {hasSSHKeys && (
           <Button
             shape="round"
             className="hidden xl:inline-block mr-3"
             disabled={status !== 'VmiReady'}
             onClick={() => setSshModal(true)}
           >
-            SSH
+            Connect via SSH
```

Comment:
How can I detect if it is a container or a vm at that point in the component tree?

Reply:
We'd need to get it from up above. You should see who calls this component and add the corresponding thing. @cannarelladev got any suggestion?

Reply:
Is it a good idea tho? We would have to drill a prop deep down in an already complex component tree just for a tooltip. Maybe it could be useful to revisit the way we pass the `instance` prop down this tree? Eventually adding a `isContainer` property to that specific object?

Reply:
I always preferred to pass down bigger objects to avoid these situations, however the initial idea for this project has been to pass only what's needed (it's actually much more practical for testing and related stuff). By my side it would not be an issue to directly pass down the whole instance. We could talk about this tomorrow maybe

Reply:
I think that passing the whole `instance` could be a good idea. But what would be the `instance` 's field that determines if this entry is a vm or a container?
```ts
export type Instance = {
  id: number;
  gui?: boolean;
  idTemplate?: string;
  templatePrettyName?: string;
  persistent?: boolean;
  tenantId?: string;
  tenantDisplayName?: string;
  tenantNamespace?: string;
  name: string;
  ip: string;
  status: VmStatus;
  url: string | null;
  timeStamp?: string;
  workspaceId?: string;
};
```